app/services/forest_liana/search_query_builder.rb

```diff
@@ -34,6 +34,16 @@ def format_column_name(table_name, column_name)
       ForestLiana::AdapterHelper.format_column_name(table_name, column_name)
     end
 
+    def acts_as_taggable_query(tagged_records)
+      ids = tagged_records
+        .map {|t| t[@resource.primary_key]}
+        .join(',')
+
+      if ids.present?
```

Comment:
On a search, If there is no `ids` found (no record tagged with the search value), the liana crashes.